app/bundle/src/main/scala/org/bitcoins/bundle/gui/NeutrinoConfigPane.scala

```diff
@@ -46,6 +49,24 @@ class NeutrinoConfigPane(
     }
   }
 
+  private val networkList = new ObservableBuffer[String]()
+
+  private val networkComboBox: ComboBox[String] = new ComboBox[String]() {
+    networkList ++= BitcoinNetworks.knownNetworks.map(_.toString)
+    items = networkList
```

Comment:
to avoid making the `ObservableBuffer` you shoud be able to do this instead
```
new ComboBox[String](BitcoinNetworks.knownNetworks.map(_.toString))
```

Reply:
I get

[info] compiling 1 Scala source to /Users/ivan/code/bitcoin-s/app/bundle/target/scala-2.13/classes ...
[error] /Users/ivan/code/bitcoin-s/app/bundle/src/main/scala/org/bitcoins/bundle/gui/NeutrinoConfigPane.scala:53:67: type mismatch;
[error]  found   : Seq[String]
[error]  required: javafx.collections.ObservableList[String]
[error]     new ObservableBuffer[String](BitcoinNetworks.knownNetworks.map(_.toString))

any hints?